Floofbot/Modules/Administration.cs

```diff
@@ -146,6 +148,45 @@ public class Administration : ModuleBase<SocketCommandContext>
             await Context.Channel.SendMessageAsync("", false, builder.Build());
         }
 
+        [Command("purge")]
+        [Alias("p")]
+        [Summary("Deletes recent messages from a given user for all channels on the server")]
+        [RequireContext(ContextType.Guild)]
+        [RequireUserPermission(GuildPermission.BanMembers)]
+        public async Task PurgeUserMessages(
+            [Summary("user")] string user)
+        {
+            IUser badUser = resolveUser(user);
```

Comment:
will this command work after a user is banned?

Reply:
All it has to do is resolve the parameter user, which is either the userid or the username#tag.
I'm thinking that if banning a user after they already left the server works, then this should work too.

Reply:
it does not, though :( I think we wont be able to resolve this until we implement a user cache